src/components/Txt/index.tsx

```diff
@@ -52,17 +64,22 @@ const BaseTxt = styled.div<TxtProps>`
 	${align}
 	${monospace}
 	${whitespace}
-
+	${code}
 	${caps}
 	${bold}
 	${italic}
-
 	${truncate}
 `;
 
 const Factory = (tag?: string) => {
 	return asRendition<React.FunctionComponent<TxtProps>>((props: any) => {
-		return <BaseTxt as={tag} {...props} />;
+		return props.copy ? (
+			<Copy content={props.copy} show={props.showCopyMode || 'always'}>
+				<BaseTxt as={tag} {...props} />
+			</Copy>
```

Comment:
Since `Copy` isn't receiving `{...props}`, its styling will be off compared to `BaseTxt`'s. So, I'm not certain but I think `{...props}` should be passed to `Copy` instead of `BaseTxt`. I believe `BaseTxt` will not be happy about no longer receiving `{...props}` so please look into that. If `BaseTxt` requires `props` to function, maybe there is a specific property in `props` that we can destructure and pass to it in order to get it to work?

Giving `Copy` `{...props}` won't be enough to get its styling working properly. I was testing by updating the `Txt.span` containing the `app.total_devices` in the UI's `ApplicationListBox` component so you can use that as your test case. You'll see that while giving `Copy` `{...props}` makes it resize and obey color and other styling, it does not center the text like it is supposed to. Please see if you can find what's causing that and how to fix it.